blog/2024-05-20-v0-format-modifications.mdx

```diff
@@ -0,0 +1,60 @@
+authors: [kenodegard, chatgpt]
+tags: [announcement, conda, conda-build, recipe, cep-13, cep-14]
+description: >
+  The recent approval of the new recipe format (`recipe.yaml`, v1) does not mean the end of the
+  old format (`meta.yaml`, v0). The conda community must balance innovation with support for
+  legacy systems.
+image: img/blog/2024-05-20-v0-format-modifications/banner.png
+---
+
+[cep-13]: https://github.com/conda/ceps/blob/main/cep-13.md
+[cep-14]: https://github.com/conda/ceps/blob/main/cep-14.md
+[rattler-build]: https://prefix-dev.github.io/rattler-build/latest/
+
+The recent approval of a new recipe format (v1) by the Conda Steering Council
+via [CEP-13][cep-13] and [CEP-14][cep-14] marks an important milestone in the
+evolution of package management in the conda ecosystem. While the community
+embraces the potential improvements that come with the new format, it's
+essential to acknowledge that the v1 format is still under development and is
+being prototyped in [rattler-build][rattler-build]. Consequently, the v0 format
+remains widely in use, emphasizing the need to balance innovation with support
+for legacy systems.
+
+<!-- truncate -->
+
+In any evolving ecosystem, such as conda, change is inevitable; this requires a
+delicate balance between progress and continuity. While the focus is on
+innovating with the v1 format the transition and adoption will take time.
+Consequently it's essential to remain responsive to the challenges and
+requirements of those who continue to rely on the v0 format for their packaging
+needs. Updates to the v0 format should be driven by the needs of the user
+community, guided by user feedback, and aimed at addressing pain points.
+
+To initiate changes to the v0 conda-build recipe format, the following workflow
+can be followed:
+
+1. Proposal: Any proposed modifications to the v0 format should also be proposed
+   in the v1 format. The rationale here being, if it's necessary for v0 it should
+   also be necessary for v1.
+2. Approval: The proposed modification should undergo the same review and
+   approval expected for the v1 format.
+3. Implementation: Once approved, the changes can be implemented in the v0
+   format, ensuring alignment with the corresponding features in the v1 format.
+
+As the community navigates the transition from v0 to v1, clear communication and
+guidance are paramount. Developers and maintainers need to be equipped with the
+resources and support necessary to understand the implications of the new format
+and make informed decisions about when and how to migrate. Additionally, robust
+documentation and educational materials will play a crucial role in facilitating
+this transition seamlessly.
+
+The approval of the v1 format represents an exciting step forward for the conda
+ecosystem. However, it's essential to recognize that change takes time, and the
+transition to the new format will be gradual. In the interim, the v0 format
```

Comment:
Might be worth mentioning https://github.com/conda-incubator/conda-recipe-manager as a way to simplify the 80% of cases?

Reply:
Yes, that's a great idea!

Reply:
Included a CRM link in https://github.com/conda/conda-dot-org/pull/190/commits/50bf324c57de9c08bbb05050cbc79faca01c57c3